Libraries/CustomComponents/NavigationExperimental/NavigationHeader.js

```diff
@@ -117,6 +120,9 @@ class NavigationHeader extends React.Component<DefaultProps, Props, any> {
     renderRightComponent: PropTypes.func,
     renderTitleComponent: PropTypes.func,
     style: View.propTypes.style,
+    leftContainerStyle: View.propTypes.style,
+    titleContainerStyle: View.propTypes.style,
+    rightContainerStyle: View.propTypes.style,    
```

Comment:
oops that's a relic from an old PR.  I'll remove them if it will get the PR merged